src/views/Onboarding/OnboardingSetup.vue

```diff
@@ -84,6 +88,19 @@ export default {
   padding: 0 !important;
   overflow: hidden;
 
+  &_description {
+    font-weight: 500;
```

Comment:
font-weight 500 is not doing anything on screen for me

there are only 2 weights, normal (not defined) and bold (700)